testssl.sh

```diff
@@ -7547,7 +7555,7 @@ get_server_certificate() {
      CERTIFICATE_LIST_ORDERING_PROBLEM=false
      if [[ "$1" =~ "tls1_3" ]]; then
           [[ $(has_server_protocol "tls1_3") -eq 1 ]] && return 1
-          if "$HAS_TLS13" && "$HAS_SIGALGS"; then
+          if "$HAS_TLS13" && "$HAS_SIGALGS" && [[ ! "$1" =~ "tls1_3_EdDSA" ]]; then
```

Comment:
Hi David,

thanks a lot for the PR!. Here e.g. and some places below I find it better not to use quotes, unless they are needed.

Cheer, Dirk

Reply:
If you could amend CHANGES.MD with a line of this feature, that would ease matters later for me.

Reply:
> thanks a lot for the PR!. Here e.g. and some places below I find it better not to use quotes, unless they are needed.

Yes, I knew that. I just seem to forget sometimes.